source/shared/net/desktop-wallet/accounts-desktop.rst

```diff
@@ -25,7 +25,9 @@ The most common tasks on an account are:
 
 In addition, you can see the following information about the selected account on the **Balance** tab:
 
--  The **Balance Total** shows the total number of CCD on the account, which includes the CCD that is available and any staked CCD or shielded CCD. It also includes CCD that is scheduled to be released to your account on a future date.
+-  The **Account Total** shows the total number of CCDs on the account, which includes the CCDs that are available and any staked CCDs or shielded CCDs. It also includes CCDs that are scheduled to be released to your account on a future date.
```

Comment:
Account total is not on the balance tap, is it? It's only on the account card on the left, when the simple view is selected.